lib/nifty_services/base_service.rb

```diff
@@ -224,7 +226,7 @@ def process_error_message_for_key(message_key, options)
       elsif message_key.is_a?(Array) && message_key.first.is_a?(Hash)
         message = message_key
       else
-        message = I18n.t("#{i18n_errors_namespace}.#{message_key}", options)
+        message = translate("#{i18n_errors_namespace}.#{message_key}", options)
```

Comment:
Good, now services can override this method to perform custom translations if needed. \o/